lyrebird/src/library.rs

```diff
@@ -229,26 +228,22 @@ impl MusicLibrary
 					}
 				}
 			)
-			.and_then
+			.map
 			(
 				|files|
 				{
-					let files = BTreeSet::from_iter(files.iter());
```

Comment:
Given the removal of this here.. which.. fair - it is expensive to do; let's change the type of `files` in the storage for this type to a BTreeSet instead. That keeps the property this was aiming for which is stable sort of the names of files within a given directory.